py/farm_ng/core/events_file_writer.py

```diff
@@ -135,10 +141,54 @@ def _increment_file_idx(self) -> None:
         """Increment the file index."""
         self._file_idx += 1
 
+    @property
+    def header_msgs(self) -> dict[str, tuple[Event, bytes]]:
+        """Return the list of header messages.
```

Comment:
```suggestion
        """Return the list of header messages.
 
```

Reply:
Adding this to pre-commit hooks in https://github.com/farm-ng/farm-ng-core/pull/201